src/pages/IndividualArticle.jsx

```diff
@@ -20,16 +22,37 @@ export default function IndividualArticle () {
                 console.log(err)
             }
         }
+
+        const getComments = async () => {
+            try {
+                const response = await newsApi.get(`/${article_id}/comments`)
+                setComments(response.data.comments.reverse())
+            }catch(err){
+                console.log(err)
+            }
+        }
         getIndividualArticle()
+        getComments()
     },[])
 
+
+
     return(
         <>
         {articleData.title === undefined ? <p className="p-2">loading...</p> : <></>}
```

Comment:
I would prefer this to use a loading state to be more explicit, it's easier to interpret what's happening as a new set of eyes